install/kubernetes/cilium/templates/hubble-relay-configmap.yaml

```diff
@@ -1,4 +1,8 @@
 {{- if .Values.hubble.relay.enabled }}
+{{- $peerSvcPort := .Values.hubble.peerService.servicePort -}}
```

Comment:
@nathanjsweet note that the condition above in `master` guard for `.Value.hubble.enabled` as well. I don't think that's an issue, but to note.